docs/source/getting-started/assessing-utility/noise.ipynb

```diff
@@ -184,7 +184,7 @@
     "    privacy_loss=dp.loss_of(epsilon=1.0),\n",
     "    split_evenly_over=10,\n",
     "    margins={\n",
-    "        (\"ILOSTAT\", ): dp.polars.Margin(public_info=\"keys\", max_partition_length=60_000_000),\n",
+    "        \"ILOSTAT\": dp.polars.Margin(public_info=\"keys\"),\n",
```

Comment:
My preference would be to keep tuple keys as the style we present in the documentation, but support strings because it's easy to forget that comma. I think in the docs having tuples in some cases and strings in others may actually be a little confusing to the user.

(Using list keys probably isn't a good option, because they're mutable, right?)

Reply:
Changed all to tuples. Once passed into the lib, they can never change, and I remember seeing pandas representing grouping keys as tuples.